docs/source/index.rst

```diff
@@ -0,0 +1,20 @@
+.. sds-data-manager documentation master file, created by
+   sphinx-quickstart on Wed Jul 26 14:32:12 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to sds-data-manager's documentation!
+============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
```

Comment:
```suggestion
* :ref:`genindex`
```

Would we expect some content to be filled in these indexes by default or do we have to do that manually? The `genindex` is empty, but the `modindex` doesn't even exist locally for me so it is a 404 when I click on it.

Reply:
Removed that whole section